<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_ryuimage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>류교수님 이미지</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pokemonimage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>선택한 포켓몬 이미지</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_ex">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_nickname">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_fight">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>470</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>전투</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
